applications/blogs/backend/src/main/resources/libs/kyanite/blogs/components/quote/dialog/.content.json

```diff
@@ -0,0 +1,50 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "generalTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "quoteText": {
```

Comment:
We could add a second RTE for sidepanel with the compact configuration: `"configuration": "kyanite/common/components/common/rte/configurations/compact"`

Reply:
If [side-panel extra request issue](https://teamds.atlassian.net/browse/WS-3150) doesn't break this case, I'll do. Need to test it.

Reply:
It worked.